src/MOI_wrapper.jl

```diff
@@ -396,12 +391,19 @@ function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <:
     if func isa MOI.SingleVariable
         return v == func.variable
     elseif func isa MOI.ScalarAffineFunction{Float64}
-        for term in func.terms
-            if term.variable_index == v
-                return true
-            end
-        end
-        return false
+        indices = [term.variable_index for term in func.terms]
+        return v in indices
```

Comment:
```suggestion
        return any(term -> v == term.variable, func.terms)
```
This will be equivalent and avoids allocating an additional `indices` array

Reply:
Same thing below in `_constraint_contains`

Reply:
fixed